src/Chapter2.hs

```diff
@@ -349,7 +375,10 @@ Implement a function that returns only the first half of a given list.
 "b"
 -}
 -- PUT THE FUNCTION TYPE IN HERE
-firstHalf l = error "firstHalf: Not implemented!"
+firstHalf :: [a] -> [a]
+firstHalf xs = fst $ splitAt middle xs
```

Comment:
Nice work on using standard functions! Though, in here, you don't need to use `splitAt`, it's enough to _take_ only the first half :wink:

Reply:
ah shoot that makes way more sense. good suggestion.

Reply:
something like `take (div (length ls) 2) ls`?

Reply:
@OxSon Yes, that's absolutely correct :+1: